kafka-connect-aws-s3/src/main/scala/io/lenses/streamreactor/connect/aws/s3/sink/config/S3FlushSettings.scala

```diff
@@ -31,7 +31,7 @@ import scala.concurrent.duration.FiniteDuration
 object S3FlushSettings {
 
   val defaultFlushSize:     Long           = 500000000L
-  val defaultFlushInterval: FiniteDuration = 3600.seconds
+  val defaultFlushInterval: FiniteDuration = 300.seconds
```

Comment:
Here is an example which got me.

5 messages on the topic and I only set the flush mount to be 3, and I didn't know why the 2 entries do not end there. Couple that with a few remarks from prospects saying the data is not sinked and they do not change the defaults should question why 1h is appropriate. Even 5mins are potentially too much for a user who tries the basic.

Reply:
I don't disagree that it makes sense for people first testing the connector, but we will need to remember to update the docs so that nobody upgrading is surprised by the change.